.env.example

```diff
@@ -1 +1,2 @@
 INSTAGRAM_ACCESS_TOKEN=
+FACEBOOK_ACCESS_TOKEN=
```

Comment:
Any way how to make it not fail without token? So one can develop without it.

Reply:
Yes - adding real access token from 1password into your local `.env` file. Including `gatsby-source-facebook` in `gatsby-config.js` makes it require valid FB access token during build step.